cclient/types/types.h

```diff
@@ -28,8 +29,8 @@ typedef struct {
 
 typedef struct {
   size_t size;
-  char* array;
-} string_array;
+  trit_array_p* array;
+} tryte_array;
```

Comment:
We should change the name too. 
It's an array containing arrays of trits :upside_down_face: 
So trit_array_array? :/

Reply:
I like that name